customers-service/src/main/resources/schema-h2.sql

```diff
@@ -1,20 +1,30 @@
 DROP TABLE pets IF EXISTS;
 DROP TABLE types IF EXISTS;
 DROP TABLE owners IF EXISTS;
+DROP TABLE photos IF EXISTS;
 
 CREATE TABLE types (
                        id   INTEGER IDENTITY PRIMARY KEY,
                        name VARCHAR(80)
 );
 CREATE INDEX types_name ON types (name);
 
+CREATE TABLE photos (
+                        id  INTEGER IDENTITY PRIMARY KEY,
+                        name VARCHAR(80),
+                        type VARCHAR(80),
+                        image LONGTEXT
```

Comment:
Not urgent, whenever possible, how was the testing issue with byte[]/longblob and string/longtext resolved?

Reply:
we can insert a byte array as text, hence LONGTEXT. When then fetching from the DB, our entity auto converts this test to a byte array.